docs/getting-started.md

```diff
@@ -20,12 +21,12 @@ Before you get started:
 
 > ⚠️ This project uses Go Modules. Due to the current state of code generation in [controller-tools](https://github.com/kubernetes-sigs/controller-tools) and [code-generator](https://github.com/kubernetes/code-generator), KUDO currently **must** be cloned into its `$GOPATH`-based location.
 
-## Installation Instructions
+## Install KUDO to your cluster
```

Comment:
Why do we need to 
1) Have a GOPATH
2) Put this repo in the gopath

to install the controller and CRDs?  I was able to do all of this from outside my gopath:

1) `git clone git@github.com:kudobuilder/kudo.git`
2) `cd kudo`
3) `make deploy | kubectl apply -f -`

Reply:
oh right, you need gopath only when you would want to compile the project. I guess for people who want to only deploy we can simplify...